presto-benchto-benchmarks/README.md

```diff
@@ -66,7 +67,8 @@ macros:
 
 With the scene set up as in the previous section, the benchmark can be run with:
 ```bash
-./mvnw clean package -pl presto-benchto-benchmarks
-java -Xmx1g -jar presto-benchto-benchmarks/target/presto-benchto-benchmarks-*-executable.jar \
+java -Xmx1g -jar benchto-driver-0.3.jar \
```

Comment:
have you tested that to see if it works? what about JDBC jar? I think it should be added to classpath.